cnx/src/widgets/pager.rs

```diff
@@ -18,14 +18,20 @@ use crate::xcb::xcb_properties_stream;
 pub struct Pager {
     active_attr: Attributes,
     inactive_attr: Attributes,
+    hidden_attr: Attributes,
 }
 
 impl Pager {
     ///  Creates a new Pager widget.
-    pub fn new(active_attr: Attributes, inactive_attr: Attributes) -> Self {
+    pub fn new(
+        active_attr: Attributes,
+        inactive_attr: Attributes,
+        hidden_attr: Attributes,
+    ) -> Self {
```

Comment:
Do you think we can create a new struct instead and pass it to new instead of passing three parameters ?

Something like this:

```
struct PagerAttributes {
active: Attributes,
inactive: Attributes,
empty: Attributes
}
```

Note that I'm finding `empty` more clearer than `hidden` here but I would be happy to hear your views on it. Also can you add documentation comments that you explained to me in the PR here ?

Reply:
I introduced the PagerAttributes struct like you suggested.

I also renamed it to `non_empty`. I think I got confused with windows and desktops. Like I would say the windows are hidden. But I see that hidden desktops are the same like inactive desktops.

Thank you for your feedback! Let me know what you think of my changes.

Reply:
Nice! I like the changes! I think once the documentation comment is fixed, this is good to go.